junit-reporting-handlebars/src/test/java/com/github/bogdanlivadariu/reporting/junit/builder/AllJunitReportsWithSuitesTest.java

```diff
@@ -0,0 +1,71 @@
+package com.github.bogdanlivadariu.reporting.junit.builder;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.xml.bind.JAXBException;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class AllJunitReportsWithSuitesTest {
+    List<String> xmlReports;
+
+    String reportPath = this.getClass().getClassLoader().getResource("valid-report-2.xml").getPath();
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/bogdan-livadariu/bootstraped-multi-test-results-report/file/2702870350/issues/source?bid=3293946&fileBranchId=3342385#l17)